Assets/Scripts/AsteroidSoundManager.cs

```diff
@@ -13,6 +13,7 @@ private void OnCollisionEnter(Collision other)
         // TODO: Check who this asteroid is colliding with
         // TODO: fix audio clip being cut off when black hole destroys the asteroid, and similarly the occasional
         //       console warning saying it can't play from disabled AudioSource
+        if (!other.gameObject.CompareTag("Player")) return;
```

Comment:
Added this cuz asteroids will be spam with collision sounds

Reply:
I think we may want to just leave this for now, for the sake of not being deafened by all the sounds, but we can tweak it closer to the end of the project work schedule.

Made a ticket for it.